src/test/test-namespace.c

```diff
@@ -133,5 +191,6 @@ int main(int argc, char *argv[]) {
 
         test_tmpdir("sys-devices-pci0000:00-0000:00:1a.0-usb3-3\\x2d1-3\\x2d1:1.0-bluetooth-hci0.device", z, zz);
 
-        return test_netns();
+        test_netns();
+        test_protect_kernel_logs();
```

Comment:
please add "return EXIT_SUCCESS" to the end, so that main() returns something clean